sass/club.scss

```diff
@@ -0,0 +1,147 @@
+
+
+// Vars
+$logo-height: 100px;
+
+// Everything else
+
+html {
+}
+
+html, body {
+    overflow: hidden;
+    height: 100%;
+    width: 100%;
+}
+
+body {
+    background-color: #c0c0c0;
+    transform: scale(1.5);
+    transform-origin: center;
```

Comment:
A teensy bit hacky -- the whole site was really small and a bump up in scaling makes it look much nicer.

An [important note from MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/zoom) on `zoom` vs. `transform`:
> [zoom CSS property] is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

Note that it's not supported in Firefox either due to it not being in the CSS standard, which is why I used `transform`.